harvest/harvest/pipelines.py

```diff
@@ -4,10 +4,42 @@
 # See: https://docs.scrapy.org/en/latest/topics/item-pipeline.html
 
 
+import logging
+
+import backoff
+import requests
+
 # useful for handling different item types with a single interface
 from itemadapter import ItemAdapter  # noqa: F401
 
+LOGGER = logging.getLogger(__name__)
+
 
 class HarvestPipeline:
+    @backoff.on_exception(
+        backoff.expo, requests.exceptions.RequestException, max_tries=5
+    )
+    def __indico(self, indico_id):
+        response = requests.get(f"https://indico.cern.ch/export/event/{indico_id}.json")
+
+        response.raise_for_status()
+        data = response.json()
+
+        sponsor = None
+        try:
+            sponsor = data["results"][0]["organizer"].split("/\\-")[0].strip()
```

Comment:
```suggestion
            sponsor = data["results"][0]["organizer"].split("-")[0].strip()
```

Reply:
The organizer sometimes has the format of "[organizer name] / [number of participants]" OR [organizer name] - [number of participants]" so both `/` and `-` have to be there.

Reply:
then neither of these will work :) you have to either use regex or check which case it is and split correctly

Reply:
Almost there you will need to do the following:

```python
sponsor = re.split("[\/|-]", data["results"][0]["organizer"])[0].strip()
```

Reply:
Okay, clear now. I use this instead then.